### PR TITLE
Argparse support with some extra features

### PR DESCRIPTION
**Pull request by GettingGodlyInThisGame**
As I can't add to the other thread (Or if I can, I can't find it) I'm just gonna make an extra one.
Simply this will add Argparse, Aswell as also adding the port as some web servers are on different ports eg Apache backend with Nginx front proxy, sockets, verbose mode (Just logging made a little bit nicer) aswell as Random User Agents which just will grab a random user agent from the array which is from the most common User Agents.

Basically: 
python3 slowloris.py --host example.com --sockets 200 --randuseragents --verbose
Randomises user agents, has logging, attacks example.com @ port 80 with 200 threads.

Feel free to edit the names such as randuseragents to useragents or shortening --verbose to -v

### CODE REVIEW

**Comment by gkbrk** [comment]
Amazing work! One question, is it possible with the argument parser to use the last argument as the host. I mean without having to do `--host example.com`.

**Comment by GettingGodlyInThisGame** [comment]
So as in running something like:
```
python3 slowloris --sockets 200 --verbose example.com
```

**Comment by GettingGodlyInThisGame** [comment]
There, hopefully that works, test that out.
Example command:
```
~/Desktop/dev/slowloris(branch:master) » python slowloris.py 192.168.1.111
Attacking 192.168.1.111 with 8 sockets.
Creating sockets...
```

**Comment by gkbrk** [comment]
@GettingGodlyInThisGame Merged, thank you for your contribution. I have done some minor edits, incremented the version number and pushed it to PyPI.

https://github.com/gkbrk/slowloris/commit/a7851b5570c5ca138c947a3f5557aff28fb826b0

If I missed anything or accidentally broke your code just let me know.
